pkg/cloud/gcp/gcs_storage.go

```diff
@@ -343,6 +345,24 @@ func (g *gcsStorage) Close() error {
 	return g.client.Close()
 }
 
+func shouldRetry(err error) bool {
+	if defaultShouldRetry(err) {
+		return true
+	}
+
+	if e := (http2.StreamError{}); errors.As(err, &e) {
+		if e.Code == http2.ErrCodeInternal {
+			return true
+		}
+	}
+
+	if e := (errors.Wrapper)(nil); errors.As(err, &e) {
```

Comment:
This looks correct but is it more intuitive to write `if ok := e.(errors.Wrapper); ok { ... }`? @knz is there a more canonical way to check if an error is wrapped?

Reply:
FYI I had to write it like this because of a checker that we have. This pattern is the suggested one:
https://github.com/cockroachdb/cockroach/blob/master/pkg/testutils/lint/passes/errcmp/errcmp.go#L97

Reply:
OH! never mind me then 👍